packages/nx/src/migrations/update-17-0-0/use-minimal-config-for-tasks-runner-options.ts

```diff
@@ -1,6 +1,20 @@
 import { updateJson } from '../../generators/utils/json';
 import { Tree } from '../../generators/tree';
 import { NxJsonConfiguration } from '../../config/nx-json';
+import { PackageJson } from '../../utils/package-json';
+import { formatChangedFilesWithPrettierIfAvailable } from '../../generators/internal-utils/format-changed-files-with-prettier-if-available';
+
+function removeNxCloudDependency(tree: Tree) {
+  if (tree.exists('package.json')) {
+    updateJson<PackageJson>(tree, 'package.json', (packageJson) => {
+      delete packageJson.dependencies?.['nx-cloud'];
+      delete packageJson.devDependencies?.['nx-cloud'];
+      delete packageJson.dependencies?.['@nrwl/nx-cloud'];
+      delete packageJson.devDependencies?.['@nrwl/nx-cloud'];
+      return packageJson;
+    });
+  }
+}
```

Comment:
nit: move below migrate function, easier to locate what's actually running if its the first thing in the file